src/ext/property-panel/settings/styling-panel/utils/create-font-family-item.ts

```diff
@@ -0,0 +1,76 @@
+import type { stardust } from "@nebula.js/stardust";
+import { DEFAULT_FONT_FAMILY } from "../../../../../pivot-table/constants";
+import type { Args, CurrentTheme } from "../../../../../types/QIX";
+import { toValueLabel } from "./to-value-label";
+
+type ThemeAccessor = (currentTheme: CurrentTheme) => string;
+
+interface Props {
+  ref: string;
+  themeAccessor: ThemeAccessor;
+  translator: stardust.Translator;
+}
+
+const DEFAULT_FONT_FAMILIES: string[] = [
```

Comment:
I saw it somewhere in some repo this: https://github.com/qlik-trial/sense-client/blob/b9115ce436c3b8e36e84a1ca9cb4283359a1917c/web/assets/client/utils/theme/theme.js#L342-L348
has been used to access font families, so probably dont we need to hardcode available font families (+ font sizes) here?

Reply:
There is no way to access that. The theme API that is exposed is coming from https://github.com/qlik-oss/nebula.js/blob/3b4b6ece98251d952eff93084bca464e2f1e531d/apis/theme/src/index.js#L32